Content.Shared/Movement/Components/InputMoverComponent.cs

```diff
@@ -72,7 +74,10 @@ public sealed partial class InputMoverComponent : Component
 
         public const float LerpTime = 1.0f;
 
-        public bool Sprinting => (HeldMoveButtons & MoveButtons.Walk) == 0x0;
+        //NOTE I don't think I'm supposed to do this
+        public bool Sprinting => IoCManager.Resolve<IConfigurationManager>().GetCVar(CCVars.GameSprintWalks)
+            ? (HeldMoveButtons & MoveButtons.Sprint) != 0x0
+            : (HeldMoveButtons & MoveButtons.Sprint) == 0x0;
```

Comment:
There's logic in my Components!

Yeah, this isn't ideal, Component should just hold the state. This is also a really goofy way of doing it. You're *also* querying the CVar every single time this is checked, which doesn't seem ideal?